recipes/vobject/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "vobject" %}
+{% set version = "0.9.3" %}
+{% set md5 = "fe5996008d5e4858d63156b37b43d59c" %}
```

Comment:
We should probably start moving to `sha256` for checksums, but this can easily be done in the feedstock.